ReactAndroid/src/main/java/com/facebook/react/animated/NativeAnimatedNodesManager.java

```diff
@@ -312,11 +316,32 @@ public void addAnimatedEventToView(int viewTag, String eventName, ReadableMap ev
     }
 
     EventAnimationDriver event = new EventAnimationDriver(pathList, (ValueAnimatedNode) node);
-    mEventDrivers.put(viewTag + eventName, event);
+    String key = viewTag + eventName;
+    if (mEventDrivers.containsKey(key)) {
+      mEventDrivers.get(key).add(event);
+    } else {
+      List<EventAnimationDriver> drivers = new ArrayList<>(1);
```

Comment:
Changed this one to an ArrayList, which makes more sense if we optimize for 1 element.